submissions/t-jay-777/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,120 @@
+						<div class="popup-menu" id="popup-menu">
+
+							<ul class="nonscroll">
+								<li>
+									<a href="#"><img src="img/calendar.png" alt="calendar"></a>
+									<span>Calendar</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/mail.png" alt="mail"></a>
+									<span>Mail</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/news.png" alt="news"></a>
+									<span>News</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/maps.png" alt="map"></a>
+									<span>Map</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/files.png" alt="files"></a>
+									<span>Files</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/contacts.png" alt="contacts"></a>
+									<span>Contacts</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/tanslation.png" alt="translate"></a>
+									<span>Translate</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/videos.png" alt="video"></a>
+									<span>Videos</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/search.png" alt="search"></a>
+									<span>Search</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/photos.png" alt="photos"></a>
+									<span>Photos</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/apps.png" alt="applications"></a>
+									<span>Apps</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/documents.png" alt="documents"></a>
+									<span>Documents</span>
+								</li>
+							</ul>
+							<input type="checkbox" class="non-scroll-more" id="more">
+							<label for="more" class="label-more"></label>
+							<ul class="scroll">
+								<li>
+									<a href="#"><img src="img/chat.png" alt="chat"></a>
+									<span>Chat</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/classroom.png" alt="classroom"></a>
+									<span>Classroom</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/mars.png" alt="mars"></a>
+									<span>Mars</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/collection.png" alt="collection"></a>
+									<span>Collection</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/blogs.png" alt="blogs"></a>
+									<span>Blogs</span>
+								</li>
+								<li>
+									<a href="#"><img src="img/ads.png" alt="ad"></a>
+									<span>Ads</span>
+								</li>
+								<a href="#" id="scroll-from">Even more from Kottans</a>
+							</ul>
+						</div>
+                    </div>
+                </li>
+                <li class="nav-submenu">
+                    <a href="#"><img src="img/bell.png" alt="notifications"></a>
+                </li>
+                <li class="nav-submenu">
+                    <a href="#"><img src="img/favicon.png" alt="user"></a>
+                </li>
+
+            </ul>
+        </div>
+    </div>
+</body>
+
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)